src/server/destination/another-destination/index.js

```diff
@@ -0,0 +1,20 @@
+import { PageController } from '../../common/controller/page-controller/page-controller.js'
+import { ExitPage } from '../../common/model/page/exit-page-model.js'
+
+/**
+ * @import { ServerRegisterPluginObject } from '@hapi/hapi'
+ */
+
+export class AnotherDestinationExitPage extends ExitPage {
+  pageTitle = 'This service is not available for your movement type'
+  urlPath = `/destination/another-destination`
```

Comment:
In the prototype, this is `/destination/can-not-use-service`.  Will update this when I do my testing sweep.

Reply:
done